index.js

```diff
@@ -0,0 +1,45 @@
+let weather = {
+"apiKey" : "4ff50176f17a1a9eec162b0203a0abad",
+fetchWeather: function(sity) {
```

Comment:
you can try async function here also its more readable if you use "try catch" something like this
`fetchWeather: async function (city) {
    try {
      const response = await fetch(
        `https://api.openweathermap.org/data/2.5/weather?q=${city}&units=metric&appid=${this.apiKey}`
      );
      const data = await response.json();
      this.displayWeather(data);
    } catch (error) {
      console.error("Error fetching weather data:", error);
    }
  },`